code-samples/oauth/simple-oauth-webserver/index.html

```diff
@@ -20,18 +20,17 @@
 			return randStr;
 		}
 
-		window.onload = () => {
-			const fragment = new URLSearchParams(window.location.hash.slice(1));
-
-			if (fragment.has("access_token")) {
-				const urlState = fragment.get("state");
+		window.onload = function () {
+            const fragment = parseQuery(window.location.href);
```

Comment:
The `parseQuery` function doesn't seem to work as intended. Here are some quick tests I did:
![](https://i.imgur.com/mGTbfZv.png)

Changing it from a `new URLSearchParams()` fragment variable would also require you to rephrase a few sentences in the paragraphs on the guide page. I think it's better to remove the custom function and keep it as it was.